src/Test/AbstractBlockServiceTestCase.php

```diff
@@ -19,13 +19,6 @@
 use Sonata\BlockBundle\Block\BlockServiceManagerInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
-/*
- * NEXT_MAJOR: remove check and extend PHPUnit\Framework\TestCase when dropping support for PHPUnit 4
- */
-if (!class_exists('\PHPUnit_Framework_TestCase')) {
-    class_alias('PHPUnit\Framework\TestCase', '\PHPUnit_Framework_TestCase');
```

Comment:
This aliases `TestCase` to the phpunit 4 version, so that extending `TestCase` works even when that class does not exist. Why remove that?

Reply:
If we want to keep compat with phpunit 4, then we might instead change on what class is extending the abstracttestcase, because now it is extending directly from TestCase

Reply:
Yeah but that provides compat with phpunit 6. So right now we might have 4,5 and 6

Reply:
`class_alias` create an alias for `PHPUnit_Framework_TestCase` which is not used on this file at the moment